tests/ui/unknown_clippy_lints.rs

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Should this test also the old `cfg_attr` attributes?

Reply:
I would recommend to explicitly write a lint for `cfg_attr`. I actually already did: https://github.com/flip1995/rust-clippy/commit/b624019848268205d7846c75392db2cf19f15230

The problem is, that it's impossible to lint crate-level `cfg_attr`s, which makes this lint only semi effective.